src/main/java/com/artipie/management/api/ApiRepoPostRtSlice.java

```diff
@@ -65,23 +64,26 @@ public Response response(
                 .thenApply(
                     form -> {
                         final Response res;
-                        final String action = ApiRepoPostRtSlice.value(form, "action");
-                        if (Action.UPDATE.value().equals(action)) {
+                        final ValueFromBody vals = new ValueFromBody(form);
+                        final Optional<String> meth = vals.byName("method");
```

Comment:
SHouldn't it be `action`? https://github.com/artipie/management-api/blob/master/src/main/resources/dashboard/repo.hbs#L46

Reply:
I missed this one. I stashed some changes and after that apply this changes but got conflict and after fixing this one missed this name